### PR TITLE
Fix a couple of CI issues due to neglect/bitrot

### PR DESCRIPTION
**Pull request by jcristau**

### CODE REVIEW

**Comment by jcristau** [comment]
I'm hoping the remaining issue with the docs fixes itself when new code lands on master and we get CI running there again; the previous push is old enough that everything's expired.

File: tests/test_version.py [hunk 1]
```diff
@@ -1,19 +1,24 @@
 import os
 import re
+import warnings
 
-from pkg_resources import get_distribution
+try:
+    from importlib.metadata import distribution
+except ImportError:
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", DeprecationWarning)
```
Comment (JohanLorenzo) [inline tests/test_version.py:9]: Question: What do these statements do?

File: tests/test_version.py [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
 
 def test_version_in_setuppy():
-    dist = get_distribution('mar')
+    dist = distribution('mar')
     assert mardor.version_str == dist.version
     assert ".".join(str(_) for _ in mardor.version) == dist.version
 
 
 def test_version_in_changelog():
-    dist = get_distribution('mar')
```
Comment (JohanLorenzo) [inline tests/test_version.py:16]: Thanks for cleaning up that unused assignment.

**Review by JohanLorenzo — APPROVED** [review]
LGTM!

File: tests/test_version.py (hunk 1 above)
Comment (jcristau) [reply tests/test_version.py:9]: We run `pytest` with `-W error`, so the pkg_resources deprecation warning turns into a test failure.  This filters out that warning so the tests can still run on python 2.7 and 3.7 where we don't have importlib.metadata.

**Pull request merged**